### PR TITLE
chore: add css changes needed for new ls data-flow

### PR DESCRIPTION
**Pull request by DariusZdroba**
### Description

Required changes to accommodate styling from the new data-flow design and addition of data-flows to AI fixes.
Related: https://github.com/snyk/snyk-ls/pull/733
### Checklist

- [ ] Tests added and all succeed
- [ ] Linted
- [ ] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### Screenshots / GIFs

_Visuals that may help the reviewer. Please add screenshots for any UI change. GIFs are most welcome!_

### CODE REVIEW

File: media/views/snykCode/suggestion/suggestionLS.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 @import '../../common/webview';
 
+.data-flow-number{
+  background-color: var(--vscode-editor-background);
+  background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.075), rgba(255, 255, 255, 0.075));
```
Comment (bastiandoetsch) [inline media/views/snykCode/suggestion/suggestionLS.scss:5]: can't this be pushed down to LS?

Comment (bastiandoetsch) [reply media/views/snykCode/suggestion/suggestionLS.scss:5]: Also, does the hardcoded gradient work on light and dark? I suppose we should use variables here :)

Comment (DariusZdroba) [reply media/views/snykCode/suggestion/suggestionLS.scss:5]: I couldn't find a variable that would work with different themes, without adding anything else to VSCode, I could add the change but on light mode, the background of the bubbles would be. I could use variables but I think they would probabbly still need to be defined in VSCode.

Comment (DariusZdroba) [reply media/views/snykCode/suggestion/suggestionLS.scss:5]: If this is still fine, I can add it to Language Server, but if we want to match the figma more, the bubbles should have the general background of the panel.
Screenshot 2024-12-13 at 11 51 53

Comment (DariusZdroba) [reply media/views/snykCode/suggestion/suggestionLS.scss:4]: I removed the `background-color` attribute and left it in LS, if we specifically don't want to agglomerate CSS files, we could replace the variable depending on the theme, but I'm not sure if there would be much difference.

**Review by ShawkyZ — APPROVED** [review]

**Pull request merged**